opensuse-stack-config.yaml

```diff
@@ -0,0 +1,9 @@
+public-buildpacks-ci-robots-branch: opensuse
```

Comment:
Why is this file called `opensuse-...` and not `opensuse42-...`.  And why is the branch not called `opensuse42`?

We refer to this name via the `--stack` option (and at the bottom of this file even have `stack-name: opensuse42`), so the file and branch names should match the stack name exactly.

Otherwise it will just be a cause of operator errors.  Using the full stack name will also allow us to start doing an `opensuse43` pipeline in parallel to the current `opensuse42` pipeline.

Same argument for the corresponding `sle` config file and branch name.

Reply:
Fixed in 3569368c and created the branches in the repo.